package.json

```diff
@@ -28,7 +28,7 @@
     "typescript": "tsc --project ./typings/tests",
     "prepublish": "npm run build",
     "lint-staged": "lint-staged",
-    "dev": "babel-node example/devServer.js"
+    "dev": "webpack-dev-server --config webpack.config.dev.js"
```

Comment:
I think we should make this completely separate, i.e. put a package.json into the sandbox folder and so on :smile: Should help us to play around with it a little more

cc @mxstbr

Reply:
Also, we don't have to do it in this PR, but the sandbox should have some kind of SSR setup

Reply:
> should have some kind of SSR setup

A great idea, but yeah, I'm not sure that I'll be able to do this in this PR

Reply:
Don't worry about the SSR stuff, let's ship this as a first iteration.